docs/sources/query/logcli/logcli-tutorial.md

```diff
@@ -0,0 +1,464 @@
+{package_size="Large", service_name="Delivery World", state="Florida"}
+{package_size="Medium", service_name="Delivery World", state="Illinois"}
+{package_size="Large", service_name="Delivery World", state="Texas"}
+{package_size="Medium", service_name="Delivery World", state="California"}
+{package_size="Medium", service_name="Delivery World", state="Texas"}
+{package_size="Small", service_name="Delivery World", state="Texas"}
+{package_size="Large", service_name="Delivery World", state="Illinois"}
+{package_size="Small", service_name="Delivery World", state="New York"}
+{package_size="Medium", service_name="Delivery World", state="New York"}
+{package_size="Large", service_name="Delivery World", state="California"}
+```
+
+We can further improve this query by adding `--analyze-labels`:
+```bash
+logcli series '{}' --analyze-labels
+```
+
+This will return a summary of the unique values for each label in our logs. The output will look similar to the following:
+
+```console
+Label Name    Unique Values  Found In Streams
+state         5              15
+package_size  3              15
+service_name  1              15
+```
+
+### Detected Fields
+
+Another useful feature of LogCLI is the ability to detect fields in your logs. This can be useful for understanding the structure of your logs and the keys that are present. This will let us detect keys which could be promoted to labels or to structured metadata. 
+
+```bash
+logcli detected-fields --since 24h '{service_name="Delivery World"}'
+```
+
+This will return a list of all the keys detected in our logs. The output will look similar to the following:
+
+```console
+label: city                     type: string            cardinality: 10
+label: detected_level           type: string            cardinality: 3
+label: note                     type: string            cardinality: 7
+label: package_id               type: string            cardinality: 20
+label: package_size_extracted   type: string            cardinality: 3
+label: package_status           type: string            cardinality: 4
+label: package_type             type: string            cardinality: 5
+label: receiver_address         type: string            cardinality: 20
+label: receiver_name            type: string            cardinality: 19
+label: sender_address           type: string            cardinality: 20
+label: sender_name              type: string            cardinality: 19
+label: state_extracted          type: string            cardinality: 5
+label: timestamp                type: string            cardinality: 20
+```
+
+You can now see why we opted to keep `package_id` in structured metadata and `package_size` as a label. Package ID has a high cardinality and is unique to each log entry, making it a good candidate for structured metadata since we potentially may need to query for it directly. Package size, on the other hand, has a low cardinality and is a good candidate for a label.
```

Comment:
```suggestion
You can now see why we opted to keep `package_id` in structured metadata and `package_size` as a label. Package ID has a high cardinality and is unique to each log entry, making it a good candidate for structured metadata since we potentially may need to query for it directly. Package size, on the other hand, has a low cardinality, making it a good candidate for a label.
```